.github/workflows/ci.yml

```diff
@@ -1,135 +0,0 @@
-name: ci
-
-on:
-  push:
-    tags:
-      - "v[0-9]+.[0-9]+.[0-9]+*"
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
-jobs:
-  lint:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 16
-      - uses: actions/setup-go@v2
-        with:
-          go-version: 1.18
-      - name: Install dependencies
-        run: |
-          npm install --global yarn
-          make setup
-          make init-dep
-      - name: Lint source code
-        run: make lint
-
-  integration-test:
-    runs-on: ubuntu-latest
-    needs: lint
-    services:
-      postgres:
-        image: postgres:12.4
-        env:
-          POSTGRES_DB: mlp
-          POSTGRES_USER: mlp
-          POSTGRES_PASSWORD: mlp
-        ports:
-          - 5432:5432
-      keto:
-        image: oryd/keto:v0.4.3-alpha.2
-        env:
-          DSN: memory
-        ports:
-          - 4466:4466
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
-        with:
-          go-version: 1.18
-      - name: Integration test
-        env:
-          DATABASE_HOST: localhost
-          DATABASE_NAME: mlp
-          DATABASE_USER: mlp
-          DATABASE_PASSWORD: mlp
-          KETO_URL: http://localhost:4466
-        run: make it-test-api-ci
-
-  publish-gojek-mlp-ui:
-    if: ${{ startsWith(github.ref, 'refs/tags/') }}
-    runs-on: ubuntu-latest
-    needs: lint
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 16
-          registry-url: https://registry.npmjs.org
-      - name: Install dependencies
-        working-directory: ./ui
-        run: yarn install
-      - name: Publish @gojek/mlp-ui library
-        working-directory: ./ui
-        run: |
-          yarn set-version-from-git
-          yarn lib publish
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-
-  publish-mlp-docker:
-    if: ${{ startsWith(github.ref, 'refs/tags/') }}
-    runs-on: ubuntu-latest
-    needs: integration-test
-    steps:
-      - uses: actions/checkout@v2
-      - name: Build and push docker image
-        uses: docker/build-push-action@v1
-        with:
-          username: ${{ github.actor }}
-          password: ${{ secrets.GHCR_TOKEN }}
-          registry: ghcr.io
```

Comment:
<img width="1215" alt="Screenshot 2022-10-31 at 6 34 53 PM-2" src="https://user-images.githubusercontent.com/23465343/198989084-cfbe6d29-49a0-4e91-9926-8ce49af9d81e.png">


How was this working previously? Can't see the published images from the homepage. Were they published elsehwere?

Reply:
They are under `gojek` organization: https://github.com/orgs/gojek/packages/container/package/mlp

We need to link the package to this repo, I'll give it a look soon.

Reply:
I granted write access to the `mlp` package from this repo, new images should now appear at the homepage too.